roles/neutronstar/templates/neutronstar_site.config

```diff
@@ -1,5 +1,6 @@
 process {
     container = '{{ neutronstar_container_path }}/{{ neutronstar_image }}'
+    executor = 'slurm'
```

Comment:
See above comment about slimming this config file right down..

Reply:
Done!